packages/usa-table/src/usa-table--sortable/usa-table--sortable.twig

```diff
@@ -119,7 +119,7 @@
     </thead>
     <tbody>
       <tr>
-        <th scope="row">Tango</th>
+        <td>Tango</td>
```

Comment:
info: Tango, Foxtrot, Hilo, and Bravo are the alphabetical sorting examples and not related to the data in the other columns, so they should `td` elements, not `th`